judge/models/profile.py

```diff
@@ -112,6 +112,10 @@ class Profile(models.Model):
                                       help_text=_('32 character base32-encoded key for TOTP'),
                                       validators=[RegexValidator('^$|^[A-Z2-7]{32}$',
                                                                  _('TOTP key must be empty or base32'))])
+    api_token = models.CharField(max_length=32, null=True, verbose_name=_('API token'),
```

Comment:
WHY IS THIS A CHARFIELD AND NOT A FERNET FIELD?

Reply:
It can't be because lookups are unsupported: https://django-fernet-fields.readthedocs.io/en/latest/#indexes-constraints-and-lookups